api/src/main/java/io/opentelemetry/metrics/DoubleCounter.java

```diff
@@ -61,10 +61,10 @@
    * <p>The value added is associated with the current {@code Context} and provided LabelSet.
    *
    * @param delta the value to add.
-   * @param labelSet the labels to be associated to this recording
+   * @param labelKeyValuePairs the labels to be associated to this recording.
    * @since 0.1.0
    */
-  void add(double delta, LabelSet labelSet);
+  void add(double delta, String... labelKeyValuePairs);
```

Comment:
Bind also needs to change.

Reply:
yep. I was going to do the bind usage as the next PR